docs/ops/cli.md

```diff
@@ -262,6 +262,8 @@ Action "run" compiles and runs a program.
      -z,--zookeeperNamespace <zookeeperNamespace>   Namespace to create the
                                                     Zookeeper sub-paths for high
                                                     availability mode
+     -nl,--nodeLabel <nodeLabelExpression>          Specify YARN node label for 
```

Comment:
I think this should only be a Yarn specific option and not a general option. Mesos and other environments don't necessarily support this.